seatunnel-e2e/seatunnel-connector-v2-e2e/connector-iotdb-e2e/src/test/java/org/apache/seatunnel/e2e/connector/iotdb/IoTDBIT.java

```diff
@@ -53,7 +55,7 @@
 import java.util.stream.Stream;
 
 @Slf4j
-public class IoTDBIT extends FlinkContainer {
+public class IoTDBIT extends TestSuiteBase implements TestResource {
```

Comment:
```suggestion
@DisabledOnContainer(value = {}, type = {EngineType.SEATUNNEL,EngineType.SPARK}, disabledReason = xxx)
public class IoTDBIT extends TestSuiteBase implements TestResource {
```

Disable Spark Engine：
There is a conflict of thrift version between IoTDB and Spark.Therefore, you need to execute rm -f $SPARK_HOME/jars/libthrift* and cp $IOTDB_HOME/lib/libthrift* $SPARK_HOME/jars/ to resolve it.


<img width="1204" alt="image" src="https://user-images.githubusercontent.com/14371345/205805046-aaa2fd16-fcdd-4b2d-9462-2c3eb8ec322e.png">

Reply:
Ok, thank you